src/ipa/rpi/controller/controller.cpp

```diff
@@ -39,6 +39,7 @@ static const std::map<std::string, Controller::HardwareConfig> HardwareConfigMap
 			.pipelineWidth = 13,
 			.statsInline = false,
 			.minPixelProcessingTime = 0s,
+			.cfeDataBufferStrided = true,
```

Comment:
Just a bit curious why being "strided" (or not) matters. Can we say anything about that somewhere? (even in the commit message)